usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -0,0 +1,116 @@
+DROP VIEW IF EXISTS award_delta_view;
+
+CREATE VIEW award_delta_view AS
```

Comment:
If we don't allow for users to search for data that is before FY2008 do we need to use the `vw_award_search.sql`? Would think that we don't need to add data from the `mv_pre2008_award_search` matview.